.github/actions/pack_and_deploy/action.yml

```diff
@@ -17,7 +17,7 @@ runs:
             startsWith(github.ref, 'refs/heads/2.') ||
             startsWith(github.ref, 'refs/tags') ) }} ; then
           sudo apt-get -y update
-          sudo apt-get install -y procmail awscli reprepro
+          sudo apt-get install -y procmail awscli reprepro rpm
```

Comment:
NB: Don't forget to revert it. It seems I [misguided you](https://github.com/tarantool/tarantool/pull/6269/commits/7a08de95c28757da4cdcea6c4a48f80af20eebac#r690787185).

Reply:
Reverted.